flac.md

```diff
@@ -113,11 +113,11 @@ Before the formal description of the stream, an overview might be helpful.
 - Individual subframes (one for each channel) are coded separately within a frame, and appear serially in the stream. In other words, the encoded audio data is NOT channel-interleaved. This reduces decoder complexity at the cost of requiring larger decode buffers. Each subframe has its own header specifying the attributes of the subframe, like prediction method and order, residual coding parameters, etc. The header is followed by the encoded audio data for that channel.
 - `FLAC` specifies a subset of itself as the Subset format. The purpose of this is to ensure that any streams encoded according to the Subset are truly "streamable", meaning that a decoder that cannot seek within the stream can still pick up in the middle of the stream and start decoding. It also makes hardware decoder implementations more practical by limiting the encoding parameters such that decoder buffer sizes and other resource requirements can be easily determined. __flac__ generates Subset streams by default unless the "--lax" command-line option is used. The Subset makes the following limitations on what may be used in the stream:
-  - The blocksize bits in the `frame header` (see [section on `frame header`](#frameheader)) must be 0001-1110. The blocksize must be <= 16384; if the sample rate is <= 48000 Hz, the blocksize must be <= 4608.
-  - The sample rate bits in the `frame header` (see [section on `frame header`](#frameheader)) must be 0001-1110.
-  - The bits-per-sample bits in the `frame header` (see [section on `frame header`](#frameheader))  must be 001-111.
-  - If the sample rate is <= 48000 Hz, the filter order in `LPC subframes` (see [section about `LPC subframes`](#subframelpc)) must be less than or equal to 12, i.e. the subframe type bits in the `subframe header` (see [section on `subframe header`](#subframeheader) may not be 101100-111111.
-  - The Rice partition order in a `Rice-coded residual section` (see [section on `Rice-coded residual section`](#residualcodingmethodpartitionedrice)) must be less than or equal to 8.
+  - The blocksize bits in the `FRAME_HEADER` (see [`FRAME_HEADER` section](#frameheader)) must be 0001-1110. The blocksize must be <= 16384; if the sample rate is <= 48000 Hz, the blocksize must be <= 4608.
```

Comment:
The multiple references to FRAME_HEADER does seem excessive, but I left it in that format as that is what is present in the original spec.

The overly wordy format of `SOMETHING` (see [`SOMETHING` section](#something) is so that it will create an anchor that is intelligible both in the md and in the rfc created by the make file.

This way, after the make file is run it will look like `SOMETHING` (see Section 10.19).